istioctl/cmd/workload.go

```diff
@@ -333,7 +333,18 @@ func createClusterEnv(wg *clientv1alpha3.WorkloadGroup, config *meshconfig.Proxy
 // the token is generated by kubectl under the workload group's namespace and service account
 // TODO: Make the following accurate when using the Kubernetes certificate signer
 func createCertsTokens(kubeClient kube.ExtendedClient, wg *clientv1alpha3.WorkloadGroup, dir string, out io.Writer) error {
-	rootCert, err := kubeClient.CoreV1().ConfigMaps(wg.Namespace).Get(context.Background(), controller.CACertNamespaceConfigMap, metav1.GetOptions{})
+	ns, err := kubeClient.CoreV1().Namespaces().Get(context.Background(), wg.Namespace, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("namespace %s was not found in cluster %s", wg.Namespace, err)
+	}
+	var caCertNamespaceConfigMapWithRevision string
+	revision := ns.Labels["istio.io/rev"]
```

Comment:
Is the `revision` label exists at the namespace level?

Reply:
it will exist only if you add it manually, so we will use the default cm in such case